ArduPlane/control_modes.cpp

```diff
@@ -161,9 +161,19 @@ void Plane::reset_control_switch()
 void Plane::autotune_start(void)
 {
     gcs().send_text(MAV_SEVERITY_INFO, "Started autotune");
-    rollController.autotune_start();
-    pitchController.autotune_start();
-    yawController.autotune_start();
+    const bool tune_roll = g2.axis_bitmask.get() & AUTOTUNE_AXIS_BITMASK_ROLL;
```

Comment:
Should use a new define/enum for plane autotune. `AUTOTUNE_AXIS_BITMASK_ROLL` is a AC auto-tune define.

Reply:
done